frontend/src/components/DownloadButton.tsx

```diff
@@ -0,0 +1,35 @@
+﻿import React from "react";
+import { Button, Icon } from "@equinor/eds-core-react";
+import { save } from "@equinor/eds-icons";
+import { ChartDataSet } from "../dto/ChartData";
+
+Icon.add({ save });
```

Comment:
This seems like a smell (modifying global state just by importing this module)

Maybe you should change the usage of `<Icon>` to have `data`? Ie, `<Icon data={save} />`.